app/controllers/subscriptions_management_controller.rb

```diff
@@ -101,6 +102,13 @@ def use_govuk_account_layout?
 
 private
 
+  def handle_one_login_hint
+    return unless params[:from] == "your-services" && !authenticated?
+
+    redirect_path = list_subscriptions_path
```

Comment:
Nitpick: probably don't need the variable due to it only being used once

Reply:
Nice - moved into the next line.